docs/disaster-recovery.md

```diff
@@ -17,91 +17,48 @@ This documentation assumes that `rbd mirroring` is set up between
 For more information on how to set up rbd mirroring, refer to
  [ceph documentation](https://docs.ceph.com/en/latest/rbd/rbd-mirroring/).
 
-## Deploy the Volume Replication CRD
+## Enable CSI-ADDONS Sidecar
 
-Volume Replication Operator is a kubernetes operator that provides common
- and reusable APIs for storage disaster recovery.
- It is based on [csi-addons/spec](https://github.com/csi-addons/spec)
- specification and can be used by any storage provider.
-
-Volume Replication Operator follows controller pattern and provides
- extended APIs for storage disaster recovery.
- The extended APIs are provided via Custom Resource Definition (CRD).
+CSI-Addons supports the Volume Replication feature that provides common
+and reusable APIs for storage disaster recovery.
+It is based on [csi-addons/spec](https://github.com/csi-addons/spec) specification
+and can be used by any storage provider.
 
 >:bulb: For more information, please refer to the
 > [volume-replication-operator](https://github.com/csi-addons/volume-replication-operator).
 
-* Deploy the `VolumeReplicationClass` CRD
+Deploy the CSI-Addons controller
 
 ```bash
-    kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/release-v0.1/config/crd/bases/replication.storage.openshift.io_volumereplicationclasses.yaml
+kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.10.0/deploy/controller/setup-controller.yaml
+```
 
- customresourcedefinition.apiextensions.k8s.io/volumereplicationclasses.replication.storage.openshift.io created
+Deploy the CSI-Addons CRD to create all related crds for volumereplication
 
+```bash
+  kubectl create -f https://raw.githubusercontent.com/csi-addons/kubernetes-csi-addons/v0.10.0/deploy/controller/crds.yaml
 ```
 
-* Deploy the `VolumeReplication` CRD
+The following crds gets created
 
 ```bash
-   kubectl create -f https://raw.githubusercontent.com/csi-addons/volume-replication-operator/release-v0.1/config/crd/bases/replication.storage.openshift.io_volumereplications.yaml
-
- customresourcedefinition.apiextensions.k8s.io/volumereplications.replication.storage.openshift.io created created
- ```
+volumereplicationclasses.replication.storage.openshift.io
+volumereplications.replication.storage.openshift.io  
+```
 
 The VolumeReplicationClass and VolumeReplication CRDs are now created.
 
->:bulb: **Note:** Use the latest available release for Volume Replication Operator.
-> See [releases](https://github.com/csi-addons/volume-replication-operator/branches)
+>:bulb: **Note:** Use the latest available release for CSI Addons Operator.
+> See [releases](https://github.com/csi-addons/kubernetes-csi-addons/branches)
```

Comment:
maybe point to the releases instead of branches?

https://github.com/csi-addons/kubernetes-csi-addons/releases